src/test/java/emissary/config/ConfigUtilTest.java

```diff
@@ -237,39 +238,39 @@ public void testMultipleConfigDirs() throws IOException, EmissaryException {
         final String cfgName2 = "emissary.chunky.Panda.cfg";
         createFileAndPopulate(configDir2, cfgName2, "BUZZ = \"BAH\"\n");
         final String origConfigDirProp = System.getProperty(CONFIG_DIR_PROPERTY);
-        System.setProperty(CONFIG_DIR_PROPERTY, configDir1.toString() + "," + configDir2.toString());
+        System.setProperty(CONFIG_DIR_PROPERTY, configDir1 + "," + configDir2);
 
         // run
         emissary.config.ConfigUtil.initialize();
 
         // assert
         final Configurator c1 = ConfigUtil.getConfigInfo(cfgName1);
-        assertNotNull("Configuration should have been found", c1);
-        assertEquals("Entry FOO is there", "BAR", c1.findStringEntry("FOO"));
+        assertNotNull(c1, "Configuration should have been found");
+        assertEquals("BAR", c1.findStringEntry("FOO"), "Entry FOO is there");
```

Comment:
I know this wasn't you, but the messages here should reflect what's happening when the test fails. This seems to reflect a success.